File: C++/Step5-1D Array/2920.cpp
//
//  2920.cpp
//  Step5-1D Array
//
//  Created by 김예빈 on 2019. 8. 30..
//  Copyright © 2019년 김예빈. All rights reserved.
//

#include <iostream>
using namespace std;

int main(int argc, const char * argv[]) {
    cin.tie(NULL);
    ios::sync_with_stdio(false);
    
    int arr[8], chk = 0, i;
    
    cin >> arr[0];
    
    for (i = 1; i < 8; i++)
    {
        cin >> arr[i];
        
        if (arr[i - 1] < arr[i])
        {
            chk++;
        } else {
            chk--;
        }
    }
    
    if (chk == 7)
        cout << "ascending";
    else if (chk == -7)
        cout << "descending";
    else
        cout << "mixed";
    
    return 0;
}

File: BaekJoon Online Judge/Swift/etc/2920.swift
//
//  2920.swift
//  Step05-1D Array
//
//  Created by Yebin Kim on 2019/11/1.
//  Copyright © 2019 김예빈. All rights reserved.
//

import Foundation

let list: [Int]! = readLine()?.split(separator: " ").map{ Int($0)! }
var asc: Bool?
var desc: Bool?

for i in 1..<7 {
    if (list[i] - list[i + 1]) < 0 {
        asc = true
    } else if (list[i] - list[i + 1]) > 0 {
        desc = true
    }
}

if let _ = asc, let _ = desc {
    print("mixed")
} else if let _ = asc {
    print("ascending")
} else if let _ = desc {
    print("descending")
}
